src/main/java/de/helmholtz/marketplace/cerebrum/controller/OrganizationController.java

```diff
@@ -0,0 +1,80 @@
+package de.helmholtz.marketplace.cerebrum.controller;
+
+import de.helmholtz.marketplace.cerebrum.entities.Organization;
+import de.helmholtz.marketplace.cerebrum.repository.OrganizationRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Since I started the developer manual a couple of weeks ago, that would be a goods place to put this. Additionally, we can add a rule to Sonar to give a warning or even an error when using wildcard imports.